templates/beta/.soliumrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "solium:all",
+  "rules": {
+    "imports-on-top": ["error"],
+    "variable-declarations": ["error"],
+    "array-declarations": ["error"],
+    "operator-whitespace": ["error"],
+    "lbrace": ["error"],
+    "mixedcase": 0,
+    "camelcase": ["error"],
+    "uppercase": 0,
+    "no-empty-blocks": ["error"],
+    "no-unused-vars": ["error"],
+    "quotes": ["error"],
+    "indentation": 0,
+    "whitespace": ["error"],
+    "deprecated-suicide": ["error"],
+    "pragma-on-top": ["error"],
+    "security/enforce-explicit-visibility": ["error"]
```

Comment:
It might be useful to include `"arg-overflow": ["error", 8]` to this list, to avoid having to change all the functions calls into multi-line calls.

Reply:
Is this the style we use? I copied the file from `voting` app. I see we only have that `arg-overflow` directive in `finance`. Should I add it to all apps? I absolutely fine with any style, but I think it should be consistent across all our repos/apps.

Reply:
The issue I have with `arg-overflow` is that there are instances where it doesn't make sense since the line is perfectly readable, and splitting into lines actually makes the code worse. I support making it 8 arguments or so as @sohkai proposes and then having a common-sense/line character count rule.

Reply:
Ok, adding that rule, also to other apps for consistency.